roles/translator/src/upstream_sv2/upstream.rs

```diff
@@ -659,18 +735,21 @@ impl ParseUpstreamMiningMessages<Downstream, NullDownstreamMiningSelector, NoRou
     ) -> Result<roles_logic_sv2::handlers::mining::SendTo<Downstream>, RolesLogicError> {
         debug!("Received NewExtendedMiningJob: {:?}", &m);
         info!("Is future job: {}\n", &m.future_job);
+        if self.is_work_selection_enabled() {
+            Ok(SendTo::None(None))
+        } else {
+            if !m.version_rolling_allowed {
+                warn!("VERSION ROLLING NOT ALLOWED IS A TODO");
```

Comment:
A github issue would be preferred.

Reply:
this has not been added I'm not sure of who is working on that or if anyone is doing it so for now I just left the warning there